core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -948,7 +955,7 @@ private FileBlockInfo generateFileBlockInfo(InodeFile file, BlockInfo blockInfo)
       throws InvalidPathException {
     FileBlockInfo fileBlockInfo = new FileBlockInfo();
     fileBlockInfo.setBlockInfo(blockInfo);
-    fileBlockInfo.setUfsLocations(new ArrayList<WorkerNetAddress>());
+    fileBlockInfo.setUfsLocations(new ArrayList<String>());
```

Comment:
This commit doesn't change the functionality, but is it intentional that we return an empty list of ufs locations when a file is in both Alluxio and the ufs?